.github/workflows/ci.yaml

```diff
@@ -70,7 +70,7 @@ jobs:
 
     # Pull Redis Cache docker image
     - name: Pulling Redis Cache image
-      run: docker pull redis:5.0.4-alpine
+      run: docker pull redis:6.0-alpine
```

Comment:
Let's also cleanup the unused image from the CI machine to save space

Reply:
This is happening on the Github Hosted agents afaik. Are you suggesting we should clean those?

Reply:
You are right, I thought the pipeline was running on our local hardware. Sorry, ignore 👍